corehq/apps/export/const.py

```diff
@@ -51,3 +51,4 @@
 
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
+MAX_EXPORTABLE_ROWS = 1000000
```

Comment:
Do we know what number we want here? Also should there be different numbers for cases and forms? I imagine that they have the same load on the DB